precompile/fee_config_manager.go

```diff
@@ -260,6 +261,7 @@ func GetStoredFeeConfig(stateDB StateDB) commontype.FeeConfig {
 		case blockGasCostStepKey:
 			feeConfig.BlockGasCostStep = new(big.Int).Set(val.Big())
 		default:
+			// this should not ever happen. keep this as panic.
```

Comment:
Can we change all of the comments that say "keep this as panic" ? This sounds more like a PR review comment explaining why it is KEPT this way than as a comment that belongs permanently in the code and explains why the code IS this way.

Reply:
changed